src/base/bittorrent/torrenthandle.cpp

```diff
@@ -74,16 +74,27 @@ const QString QB_EXT {QStringLiteral(".!qB")};
 
 using namespace BitTorrent;
 
+#if (LIBTORRENT_VERSION_NUM >= 10200)
+namespace libtorrent
+{
+    namespace aux
+    {
+        inline uint qHash(const strong_typedef<std::int32_t, file_index_tag> &key, uint seed)
```

Comment:
1. Personally I would drop the inline word and let compiler make the decision.
2. Why not just use `lt::file_index_t` instead of `strong_typedef<>`? this is for libt 1.2 only.
3. nitpick, `const uint seed`.

Reply:
>this is for libt 1.2 only.

Both `lt::file_index_t` and `strong_typedef<>` are from libt 1.2 only.
>Why not just use `lt::file_index_t`

It makes more unclear why the function is declared in `libtorrent::aux` namespace.